### ISSUE TITLE
Image IMU data

### ISSUE THREAD

**Issue by ShuksanGeomatics** [MobileRoboticsSkoltech/OpenCamera-Sensors#38]
I have OpenCamera-Sensors installed on a Moto Z3.  I can take a picture with the camera and read the EXIF and XMP tags but I do not see the IMU values stored there.  I see from README.MD that IMU for video is stored in the DCIM folder as a CSV.  This appears to not be the case for images.  Are image IMU values not supported?

**Comment by azaat** [comment]
Indeed, we didn't include the feature of getting IMU data when taking a picture. For now, as a temporary solution, you could record a short video and get some frame from it (with ffmpeg). Could you please provide some usecase, an example where such feature would be needed?
